package.json

```diff
@@ -72,7 +72,7 @@
     "ts-morph": "^19.0.0",
     "typescript": "^5.0.2",
     "vite": "^4.0.0",
-    "vitepress": "^1.0.0-beta.2"
+    "vitepress": "^1.0.0-rc.4"
```

Comment:
This isn’t strictly necessary (we could have upgraded vitepress with just `yarn upgrade` and changing the yarn.lock file. But I guess since we’re using the sitemap feature added in beta.7, this is okay. The point being, `1.0.0-rc.4` is already consistent with `^1.0.0-beta.2`.